cmd/sti/main.go

```diff
@@ -167,10 +168,19 @@ func newCmdBuild(cfg *api.Config) *cobra.Command {
 				glog.V(1).Infof(message)
 			}
 
+			if cfg.RunImage {
+				runner, rerr := run.New(cfg)
+				if rerr == nil {
+					rerr = runner.Run(cfg)
+				}
+				checkErr(rerr)
+			}
+
 		},
 	}
 
 	buildCmd.Flags().BoolVarP(&(cfg.Quiet), "quiet", "q", false, "Operate quietly. Suppress all non-error output.")
+	buildCmd.Flags().BoolVar(&(cfg.RunImage), "run", false, "Run resulting image as part of invocation of this command.  You may have to Ctrl-C to exit the sit command invocation and the container launched because this options was specified.")
```

Comment:
Sorry, was distracted with the other comments, missed this one.  Part of next push.